net-irc/atheme-services/files/atheme-services.initd

```diff
@@ -0,0 +1,36 @@
+#!/sbin/openrc-run
+# Copyright 1999-2011 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+extra_started_commands="reload"
+
+depend() {
+	need net
+	use ircd
+	provide irc-services
```

Comment:
I don't think it's really useful to have a 'virtual' for irc-services. But I'll ask around.